.ci/publish-images.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+BASE_BUILD_IMAGE=${BASE_BUILD_IMAGE:-"opentelemetry/opentelemetry-operator"}
+OPERATOR_VERSION=${OPERATOR_VERSION:-$(git describe --tags)}
+
+## if we are on a release tag, let's extract the version number
+## the other possible value, currently, is 'master' (or another branch name)
+## if we are not running in the CI, it fallsback to the `git describe` above
+if [[ $OPERATOR_VERSION == v* ]]; then
+    OPERATOR_VERSION=$(echo ${OPERATOR_VERSION} | grep -Po "([\d\.]+)")
+    MAJOR_MINOR=$(echo ${OPERATOR_VERSION} | awk -F. '{print $1"."$2}')
+fi
+
+BUILD_IMAGE=${BUILD_IMAGE:-"${BASE_BUILD_IMAGE}:${OPERATOR_VERSION}"}
+
+echo "Building image ${BUILD_IMAGE}"
+make docker-build IMG="${BUILD_IMAGE}"
+
+## push to Docker Hub
+if [ "${DOCKER_PASSWORD}x" != "x" -a "${DOCKER_USERNAME}x" != "x" ]; then
+    echo "Performing a 'docker login'"
+    echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
+
+    echo "Pushing ${BUILD_IMAGE}"
+    docker push "${BUILD_IMAGE}"
+
+    if [ "${MAJOR_MINOR}x" != "x" ]; then
```

Comment:
May want to pull these lines above the docker hub if, as the value is also used for quay - in case only quay credentials are provided?